packages/visualizations/src/components/Table/store.ts

```diff
@@ -1,7 +1,10 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
-import { writable } from 'svelte/store';
+import { writable, get } from 'svelte/store';
 
 const defaultLocale = navigator.language;
+const defaultDirection = 'ltr';
 
 // eslint-disable-next-line import/prefer-default-export
 export const locale = writable<string>(defaultLocale);
+export const direction = writable<string>(defaultDirection);
+export const isRtl = writable<boolean>(get(direction) === 'rtl');
```

Comment:
Only one is needed. If we really need both, then `isRtl` is a `derived`. But we're not using `direction` anywhere.